examples/favorites_advanced/getx_favorites_advanced/lib/main.dart

```diff
@@ -0,0 +1,115 @@
+import 'package:flutter/material.dart';
+import 'package:get/get_navigation/src/root/get_material_app.dart';
+import 'package:get/get_navigation/src/routes/get_route.dart';
+
+import 'feature_home/views/home_page.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return GetMaterialApp(
+      initialRoute: '/',
+      title: 'Puppies App',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      getPages: [
+        GetPage(name: '/',
+        page: () =>  HomePage(),),
+        // GetPage(name:'/puppyDetails',
+        // page: () =>  PuppyDetailsPage()),
+        // GetPage(name: '/puppyEdit',
+        //   page: () =>  PuppyEditPage(),)
+      ],
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
```

Comment:
@RomanovaPrime, I think you can remove this widget as it's not used anywhere.

Reply:
Removed